src/tools/sdk/index.md

```diff
@@ -29,8 +29,11 @@ directory that has these command-line tools:
 
 <div class="row">
   <div class="col-lg-6" markdown="1">  
-    
-  [dart](/tools/dart-vm)
+
+  [dart](/tools/dart-tool)
+  : The Dart command line tool
+
+  [dart-vm](/tools/dart-vm)
```

Comment:
Good catch! Actually, why don't we just delete the entry for the standalone VM? People can get to it from the dart-tool page.

Reply:
done, i think. i haven't worked in Git for a while.